Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
 version = "1.11.0"
 
 [deps]
+Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
```

Comment:
Note we can remove the dependency on LazyArtifacts now but doing so interferes with running PkgBenchmarks. I'll leave in this dependency for now and remove it in a follow up PR where benchmarks don't matter

Reply:
https://github.com/JuliaTime/TimeZones.jl/issues/443